src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import PeopleTable from './PeopleTable';
+import ErrorMessage from './ErrorMessage';
+import NotPeople from './NotPeople';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    let isMounted = true;
+
+    setIsLoading(true);
+
+    getPeople()
+      .then(data => {
+        if (isMounted) {
+          setPeople(data);
+        }
+      })
+      .catch(() => {
+        if (isMounted) {
+          setIsError(true);
+        }
```

Comment:
When catching an error, it might be useful to log the error or display a more informative message to the user, rather than just setting 'isError' to true. This can help with debugging and user experience.